storage/storage.rs

```diff
@@ -187,10 +188,18 @@ impl<D> MVCCStorage<D> {
         // FIXME proper error
         let mut durability_service = D::recover(path.join(Self::WAL_DIR_NAME)).expect("Could not create WAL directory");
         durability_service.register_record_type::<CommitRecord>();
+        durability_service.register_record_type::<StatusRecord>();
 
         let name = name.as_ref();
         let (keyspaces, keyspaces_index) = recover_keyspaces::<KS>(&storage_dir)?;
-        let isolation_manager = IsolationManager::new(SequenceNumber::from(1));
+
+        let isolation_manager = if durability_service.previous() == SequenceNumber::MIN {
```

Comment:
Let's add a new method called `is_empty()`, which uses the next sequence number internally and checks if it is the first allowed one. However, this leads to: in`WAL::open_from` which calls `.unwrap_or(1);` -- this should be the same constant (important constants like this should make us uncomfortable). Then it's all 'safe'.